### DISCUSSION TITLE
Hey guys I am having an issue installing the Avalonia templates. "Avalonia.Templates could not be installed, the package does not exist."

### DISCUSSION THREAD

**Discussion by kwc81** [AvaloniaUI/Avalonia.Templates#258] [Q&A]
Hey guys I am having an issue installing the Avalonia templates. "Avalonia.Templates could not be installed, the package does not exist."
Has anyone else had this issue? I have installed the latest version of .net 8 and still having this same issue.

![image](https://github.com/AvaloniaUI/avalonia-dotnet-templates/assets/45573902/65689802-8de4-4366-8f55-06b6d4c009df)

**Comment by Mrxx99** [comment]
Do you have changed something in your nuget.config?
Also can you try running it in a folder that is not a system folder?

**Reply by kwc81** [reply]
No I have not changed anything and I also tried running in outside the system folder still no luck
Can you help Id love to get started testing the avalonia ui for a new project.

**Reply by kwc81** [reply]
Is there something simple I am missing here. I am following these instructions here...https://docs.avaloniaui.net/docs/get-started/install.

Should be just a simple copy and paste of each command into power shell correct?

**Reply by Mrxx99** [reply]
yes that should work, unfortunately I don't know how to help here as I don't know and can't reproduce that error,
you can maybe run `dotnet --info` in the same folder you are running the install command, and if you are not on the latest patch version of .NET 8 you can try installing the latest. If this does not help you can clone the template repository and run the `install-dev-templates.ps1` script. Also if you are using Visual Studio you can install the [Avalonia Template Studio](https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.TemplateStudioForAvalonia) which gives you a template wizard in Visual Studio

**Comment by vittoso** [comment]
Hi @kwc81 I have exacly the same problem in my Windows 11 machine. I've tried to lauch the command in my personal c:\User\<my user> folder but without success either

**Comment by vittoso** [comment]
I fixed it... I am on a new PC and for some reason I didn't have the NuGet.org feed configured. I manually added Nuget.org https://www.nuget.org/api/v2/ into Nuget options packages source in Visual Studio 2022 and then  the  dotnet new install Avalonia.Templates command worked.

All credits to [this ](https://stackoverflow.com/questions/44508253/dotnet-new-install-is-looking-at-wrong-nuget-server)SO answer.

**Reply by axi82** [reply]
Same issue, new laptop, fresh install of VS2022, no nuget feed configured. Thanks.

**Reply by rp1231** [reply]
I had the same issue as well

**Reply by 0xVe1L** [reply]
thank you for this! I had the same issue. They should probably include this in the installation instructions
